concourse/pipelines/create-deployer.yml

```diff
@@ -413,6 +413,7 @@ jobs:
     serial: true
     plan:
     - get: paas-cf
+      passed: [vpc]
```

Comment:
See commit message & http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule

We got hit by that issue during deployment (it was preventing us doing our work), so we fixed it.